docs/tsconfig.json

```diff
@@ -1,11 +1,14 @@
 {
   "compilerOptions": {
+    "target": "ES5",
     "lib": ["es2017"],
     "moduleResolution": "node",
     "module": "commonjs",
-    "baseUrl": "."
+    "baseUrl": ".",
+    "experimentalDecorators": true
```

Comment:
Is this supported by TypeDoc?

Reply:
Unsure, will take a look.

Reply:
As far as I can tell, there is no official support for decorators in TypeDoc at the moment. However, as the implementation is just a function, there should be no issue with it showing up in the TypeDoc output. We can add information in the comment block about how it is called, as well.